### PR TITLE
c8d: show the real image creation date when listing images

### PR DESCRIPTION
**Pull request by rumpl**
**- What I did**

Used the creation date of the imge from the config and not from the containerd metadata,

```diff
$ docker pull busybox
$ docker images      
REPOSITORY   TAG       IMAGE ID       CREATED        SIZE
-busybox      latest    3fbc63216742   20 seconds ago   6.61MB
+busybox      latest    3fbc63216742   3 months ago   6.61MB
```

Also set the `ParentID` to whatever we find in the image labels

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: daemon/containerd/image_list.go [hunk 1]
```diff
@@ -276,10 +279,15 @@ func (i *ImageService) singlePlatformImage(ctx context.Context, contentStore con
 		return nil, nil, err
 	}
 
+	created := int64(0)
```
Comment (vvoland) [inline daemon/containerd/image_list.go:282]: Maybe we could leave the `rawImg.CreatedAt.Unix()` as a fallback in case the `cfg.Created` is nil?

Comment (rumpl) [reply daemon/containerd/image_list.go:282]: I'm sure @thaJeztah can link you some issue here or in the cli about what should we do with the images that don't have a creation date. It's a real thing, especially for reproducible builds so I think we should just set `0` if there is no creation date

**Review by vvoland — APPROVED** [review]

File: daemon/containerd/image_list.go (hunk 1 above)
Comment (thaJeztah) [reply daemon/containerd/image_list.go:282]: There was some discussion around dates and reproducibility in https://github.com/moby/moby/pull/45504#discussion_r1189110803 (not sure if relevant in any way)

File: daemon/containerd/image_list.go [hunk 2]
```diff
@@ -276,10 +279,15 @@ func (i *ImageService) singlePlatformImage(ctx context.Context, contentStore con
...
+	if cfg.Created != nil {
+		created = cfg.Created.Unix()
```
Comment (thaJeztah) [inline daemon/containerd/image_list.go:284]: Micro-nit; as we already have a `summary` var in which we store the image-data, I guess we can avoid the `created` var if we put this block after it;


```go
if cfg.Created != nil {
    summary.Created = cfg.Created.Unix()
}
```

Would there be a need to do ` && !cfg.Created.IsZero()` as well? (not sure)

Comment (rumpl) [reply daemon/containerd/image_list.go:284]: I think it's fine to set the `Created` in the `IsZero` case, that's actually what the GD case does

Comment (thaJeztah) [reply daemon/containerd/image_list.go]: Oh! We can; but that's fine for a separate PR, as that's not a new change, right?

My comment was mostly; if we want to; let's do so before merging, to reduce (ever so little) code churn 👍

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**